src/Control/Effect/Simple.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{- | An effect that promotes a simple higher-order function into a first-order effect, trading efficiency for simplicity.
+
+For more information about the tradeoffs associated with this formulation of effects, please see "Control.Carrier.Simple".
+
+Predefined carriers:
+
+* "Control.Carrier.Simple.SimpleC"
+
+@since 1.2.0.0
+-}
+
+module Control.Effect.Simple
+( -- * Simple effect
+  Simple (..)
+, sendSimple
+, sendSimple2
+, sendSimple3
+  -- * Re-exports
+, Algebra
+, Has
+, run
+) where
+
+import Control.Algebra
+import Data.Kind (Type)
+
+data Simple e (m :: Type -> Type) a where
+  Simple :: e a -> Simple e m a
+
+-- | Invoke an effect constructor. Can be composed with 'Prelude.(.)' to wrap constructors of one argument.
+sendSimple :: Has (Simple eff) sig m => eff a -> m a
+sendSimple = send . Simple
```

Comment:
What do you think about `send`? We can re-export whatever we need to avoid simple effect modules having to import `Control.Algebra` & qualify.